src/Microsoft.Health.TaskManagement/JobHosting.cs

```diff
@@ -39,6 +39,13 @@ public JobHosting(IQueueClient queueClient, IJobFactory jobFactory, ILogger<JobH
 
         public double JobHeartbeatIntervalInSeconds { get; set; } = Constants.DefaultJobHeartbeatIntervalInSeconds;
 
+        // TODO: Temporary method to prevent build breaks within Health PaaS. Will be removed after code is updated in Health PaaS
```

Comment:
Just a reminder, convert this to
```suggestion
        [Obsolete("Temporary method to prevent build breaks within Health PaaS. Will be removed after code is updated in Health PaaS")]
```

Reply:
Thanks. I'll make a separate PR for this change so we can at least get these in.